pkg/controller/kogitoapp/resource/deployment_config.go

```diff
@@ -39,15 +39,16 @@ const (
 	// ServiceAccountName is the name of service account used by Kogito Services Runtimes
 	ServiceAccountName = "kogito-service-viewer"
 
-	envVarInfinispanServerList     = "SERVER_LIST"
-	envVarInfinispanUseAuth        = "USE_AUTH"
+	appPropInfinispanServerList    = "INFINISPAN_SERVER_LIST"
```

Comment:
These properties serve like keys only, their value doesn't have any relevance, right?
Have you considered using https://yourbasic.org/golang/iota/ for their value? To hint code readers that the value is not relevant?